cmd/mattermost/commands/config.go

```diff
@@ -275,7 +276,15 @@ func configMigrateCmdF(command *cobra.Command, args []string) error {
 	from := args[0]
 	to := args[1]
 
-	err := config.Migrate(from, to)
+	cwd, err := os.Getwd()
+
+	if err != nil {
+		return errors.Wrap(err, "failed to migrate config")
+	}
+
+	utils.InitTranslationsWithDir(path.Join(cwd, "i18n"))
```

Comment:
As the `mattermost` command is always going to run from the server's directory, we can use the `TranslationsPreInit` function, for simplicity:

```suggestion
	if err := utils.TranslationsPreInit(); err != nil {
		return errors.Wrap(err, "failed to load translations while migrating config")
	}
```